wsmaster/che-core-api-installer/src/main/java/org/eclipse/che/api/installer/server/InstallerRegistry.java

```diff
@@ -28,12 +30,47 @@
 public interface InstallerRegistry {
 
     /**
+     * Adds installer to the registry.
+     *
+     * @param installer
+     *      the installer to add
+     * @throws InstallerConflictException
```

Comment:
WDYT about making this exception more concrete. Not it is an analog of rest ConfictException where conflict is related to `409` response code. Mb it would be better to use here exception like `InstallerAlreadyExistException`?